digits/frameworks/framework.py

```diff
@@ -48,7 +48,7 @@ def get_network_from_desc(self, network_desc):
         """
         raise NotImplementedError('Please implement me')
 
-    def get_network_from_previous(self, previous_network):
+    def get_network_from_previous(self, previous_network, useSameDataset):
```

Comment:
Seems weird to have `previous_network` (with an underscore) right next to `useSameDataset` (using camelCase).